src/neo/Ledger/Blockchain.cs

```diff
@@ -87,16 +87,23 @@ static Blockchain()
         {
             GenesisBlock.RebuildMerkleRoot();
 
-            NativeContract[] contracts = { NativeContract.GAS, NativeContract.NEO };
             using (ScriptBuilder sb = new ScriptBuilder())
             {
-                foreach (NativeContract contract in contracts)
+                foreach (NativeContract contract in new NativeContract[] { NativeContract.GAS, NativeContract.NEO })
                 {
                     sb.EmitAppCall(contract.Hash, "onPersist");
                     sb.Emit(OpCode.DROP);
                 }
-
-                onPersistNativeContractScript = sb.ToArray();
+                onPersistScript = sb.ToArray();
+            }
+            using (ScriptBuilder sb = new ScriptBuilder())
+            {
+                foreach (NativeContract contract in new NativeContract[] { NativeContract.Oracle })
+                {
+                    sb.EmitAppCall(contract.Hash, "postPersist");
+                    sb.Emit(OpCode.DROP);
```

Comment:
`postPersist` return `void` why this `DROP`?

Reply:
Because `postPersist`is a public method, and the VM push `null` to the stack automatically.